.travis.yml

```diff
@@ -1,12 +1,13 @@
-language: node_js
-node_js:
-- '0.10'
+language: dart
+dart:
+- stable
+with_content_shell: true
 before_install:
 - npm install -g bower
 - npm install -g grunt-cli
-install:
-- ./init.sh
-script:
+- npm install
+- bower install
+before_script:
```

Comment:
Setting up CI for a pure dart repo should just "work" with Travis.  The problem we are having here is because we have a js/dart hybrid repo.